modules/dnn/src/dnn.cpp

```diff
@@ -2102,7 +2102,7 @@ struct Net::Impl
 
                     auto ieInpNode = inputNodes[i].dynamicCast<InfEngineNgraphNode>();
                     CV_Assert(oid < ieInpNode->node->get_output_size());
-#if INF_ENGINE_VER_MAJOR_GT(2020020000)
+#if INF_ENGINE_VER_MAJOR_GT(2020030000)
```

Comment:
Why not `INF_ENGINE_VER_MAJOR_GE(2020030000)`?

Reply:
2020.3 still provides old API (at this moment).